pom.xml

```diff
@@ -651,6 +651,12 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>java-hamcrest</artifactId>
```

Comment:
I would recommend to stick to the 1.3 version we are already using (transitively from awaitility). Or fully migrate to 2.0. Having two versions in a same project often leads to issues